### PR TITLE
New adapter: Smrtconnect

### PR DESCRIPTION
**Pull request by psmrt**
Added Smrtconnect adapter

### CODE REVIEW

File: static/bidder-info/smrtconnect.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+endpoint: "https://amp.smrtconnect.com/openrtb2/auction?supply_id={{.SupplyId}}"
+endpointCompression: gzip
+maintainer:
+  email: "tech@smrtconnect.com"
```
Comment (Sonali-More-Xandr) [inline static/bidder-info/smrtconnect.yaml:4]: @psmrt We tried sending a verification mail to tech@smrtconnect.com but the mail bounced. Could you please check if this is the correct address and it is working

Comment (Sonali-More-Xandr) [reply static/bidder-info/smrtconnect.yaml]: Thank you @psmrt . We have sent an email to the updated address. Please reply to the mail with "Received" to complete the verification process.

Comment (psmrt) [reply static/bidder-info/smrtconnect.yaml]: done

**Comment by psmrt** [comment]
java adapter link -> https://github.com/prebid/prebid-server-java/pull/3060

File: adapters/smrtconnect/smrtconnect.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+	}
+
+	bidder := &adapter{
+		endpoint: template,
+	}
+	return bidder, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	smrtconnectExt, err := getImpressionExt(&request.Imp[0])
```
Comment (gargcreation1992) [inline adapters/smrtconnect/smrtconnect.go:34]: Just curious - Why are you considering just the first impression and ignoring others. Should this be in the for loop?

Comment (gargcreation1992) [reply adapters/smrtconnect/smrtconnect.go:34]: @psmrt Requesting you to please address this comment.

File: adapters/smrtconnect/smrtconnect.go [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+		endpoint: template,
+	}
+	return bidder, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var requests []*adapters.RequestData
+
+	requestCopy := *request
+	for _, imp := range request.Imp {
```
Comment (gargcreation1992) [inline adapters/smrtconnect/smrtconnect.go:37]: Requesting you to please add a test case with request having multiple impressions.

**Review by gargcreation1992 — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**